usagov/vote-gov-drupal
# Feature/vote 1939 voter guide refinements

## Jira ticket

[Vote-1939](https://cm-jira.usa.gov/browse/VOTE-1939)

## Description

Updates to the voter guide template from PX review 

Updates made:
- Updated page masthead to have hero mark-up 
- Updated the  hero style sheets for the `vote-hero--dark` to help match the style guide 
	- mobile view for image 
	- breadcrumb update 
	- background color 
- added side page nav and fixed bug with the active state 
- added a static image

Follow up tasks:
These have been ticketed out 
- migrate hero paragraph type to be a component 
- update feature card to be a view 
- update space between breadcrumb and header 
- update image to be dynamic 

## Deployment and testing

### Post-deploy steps

1. run `lando retune` and cd into `votegov` theme and run npm run build 

### QA/Testing instructions

1. visit voter guide page and compare to [design here](https://www.figma.com/design/u82ICmmHNBKpE1Tw4nsO0K/Vote.gov-Redesign-2024-(Master)?node-id=9046-27784&m=dev)
2. check on various browsers and refer to ticketed out work to check if something is off to make sure it will be addressed later on 

## Checklist for the Developer




## Checklist for the Peer Reviewers
